eva_submission/ingestion_templates.py

```diff
@@ -89,4 +89,5 @@ def variant_load_props_template(
         'annotation.skip': annotation_skip,
         'annotation.overwrite': False,
         'config.chunk.size': 200,
+        'spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation': True,
```

Comment:
What is this doing ?

Reply:
It just silences the `java.lang.reflect.InvocationTargetException` that shows up at the beginning of all the pipeline logs since the Spring 2 upgrade - it's not a real error, just an annoyance.